### DISCUSSION TITLE
Unable to Connect to Frontier Api

### DISCUSSION THREAD

**Discussion by emyrsmorgan** [EDCD/EDDI#2573] [EDDI Standalone Q&A]
When i click connect to Frontier api in Version 4.0.3 Eddei pauses for a few seconds then crashes

**Comment by Tkael** [comment]
Please create a bug report here on Github and please attach a `.zip` file of your `.log` file from an instance when this crash occurred (`.log` files are found in `%appdata%/EDDI`). If you're not sure which `.log` contains the exception / crash then please include all of them. o7

**Reply by emyrsmorgan** [reply]
done
